doc/releases/changelog-dev.md

```diff
@@ -272,94 +275,106 @@
+  [(#2807)](https://github.com/PennyLaneAI/pennylane/pull/2807)
+  [(#2849)](https://github.com/PennyLaneAI/pennylane/pull/2849)
+  
+  Many quality-of-life features have been added to how we handle Operators. This
+  includes:
 
-  The `Sum` class provides functionality like any other PennyLane operator. We can
-  get the matrix, eigenvalues, terms, diagonalizing gates and more.
+  - Add Operators via `qml.op_sum` to create a new Operator whose matrix, terms, 
+  and eigenvalues can be accessed as per usual.
 
   ```pycon
-  >>> summed_op = qml.op_sum(qml.PauliX(0), qml.PauliZ(0))
+  >>> summed_op = qml.op_sum(qml.PauliX(0), qml.PauliZ(1))
   >>> summed_op
-  PauliX(wires=[0]) + PauliZ(wires=[0])
+  PauliX(wires=[0]) + PauliZ(wires=[1])
   >>> qml.matrix(summed_op)
-  array([[ 1,  1],
-         [ 1, -1]])
+  array([[ 1,  0,  1,  0],
+         [ 0, -1,  0,  1],
+         [ 1,  0,  1,  0],
+         [ 0,  1,  0, -1]]) 
   >>> summed_op.terms()
-  ([1.0, 1.0], (PauliX(wires=[0]), PauliZ(wires=[0])))
+  ([1.0, 1.0], (PauliX(wires=[0]), PauliZ(wires=[1])))
   ```
 
-  The `summed_op` can also be used inside a `qnode` as an observable.
-  If the circuit is parameterized, then we can also differentiate through the
-  sum observable.
+  Summed Operators can also be used inside of a QNode as observables while maintaining
+  differentiability:
 
   ```python
-  sum_op = Sum(qml.PauliX(0), qml.PauliZ(1))
+  summed_op = qml.op_sum(qml.PauliX(0), qml.PauliZ(1))
+
   dev = qml.device("default.qubit", wires=2)
 
-  @qml.qnode(dev, grad_method="best")
+  @qml.qnode(dev)
   def circuit(weights):
-        qml.RX(weights[0], wires=0)
-        qml.RY(weights[1], wires=1)
-        qml.CNOT(wires=[0, 1])
-        qml.RX(weights[2], wires=1)
-        return qml.expval(sum_op)
+      qml.RX(weights[0], wires=0)
+      qml.RY(weights[1], wires=1)
+      qml.CNOT(wires=[0, 1])
+      qml.RX(weights[2], wires=1)
+      return qml.expval(summed_op)
   ```
 
   ```pycon
   >>> weights = qnp.array([0.1, 0.2, 0.3], requires_grad=True)
   >>> qml.grad(circuit)(weights)
-  tensor([-0.09347337, -0.18884787, -0.28818254], requires_grad=True)
+  array([-0.09347337, -0.18884787, -0.28818254])
   ```
 
-* Added `__add__` and `__pow__` dunder methods to the `qml.operation.Operator` class so that users can combine operators
-  more naturally. [(#2807)](https://github.com/PennyLaneAI/pennylane/pull/2807)
-
-  ```python
-  >>> summed_op = qml.RX(phi=1.23, wires=0) + qml.RZ(phi=3.14, wires=0)
-  >>> summed_op
-  RX(1.23, wires=[0]) + RZ(3.14, wires=[0])
-  >>> exp_op = qml.RZ(1.0, wires=0) ** 2
-  >>> exp_op
-  RZ**2(1.0, wires=[0])
-  ```
-
-* Added support for addition of operators and scalars. [(#2849)](https://github.com/PennyLaneAI/pennylane/pull/2849)
+  - You can now add scalars to Operators, where the interpretation is that the 
+  scalar is a properly-sized identity matrix. 
 
   ```pycon
   >>> sum_op = 5 + qml.PauliX(0)
```

Comment:
Why doesn't this work?

```
sum_op = 5 + qml.CNOT(wires=[0,1])
```

The error is: `ValueError: Identity: wrong number of wires. 2 wires given, 1 expected.`

Is this feature not generalizable to multi-qubit gates?